internal/service/grafana/resource_service_grafana_test.go

```diff
@@ -172,6 +174,7 @@ func TestAccAivenService_grafana_with_ip_filter_objects(t *testing.T) {
 	})
 }
 
+//lint:ignore U1000 This test is skipped
```

Comment:
could we use `//nolint:unused` instead, with no comment also, for consistency across the project?